src/DataAccess/Repositories/EarningsQueryRepository.cs

```diff
@@ -12,6 +12,9 @@ public class EarningsQueryRepository : IEarningsQueryRepository
         private readonly Lazy<ApprenticeshipEarningsDataContext> _lazyContext;
         private ApprenticeshipEarningsDataContext DbContext => _lazyContext.Value;
 
+        private const decimal GovernmentContribution = 0.95m;
```

Comment:
This ratio feels pretty crucial and so may potentially need to be used elsewhere or edited at a later date (correct me if I'm wrong) - should we consider storing this somewhere separately where core metrics/constants live?

Reply:
We did have a discussion around this with the wider team at the time - "Should it be configurable?" The consensus was that if we were to change this contribution percentage it would have impacts everywhere and an impact analysis would have to be done and we shouldn't spend time now worrying about it. That being said it could maybe live in at least a constants file, will update.